deps/build.jl

```diff
@@ -9,7 +9,7 @@ else
 
     provides(Binaries, URI("http://bitbucket.org/MDukhan/yeppp/downloads/yeppp-$ver.tar.bz2"), libyepppdep, unpacked_dir = "yeppp-$ver/binaries/windows/amd64/", os = :Windows)
     provides(Binaries, URI("http://bitbucket.org/MDukhan/yeppp/downloads/yeppp-$ver.tar.bz2"), libyepppdep, unpacked_dir = "yeppp-$ver/binaries/macosx/x86_64/", os = :Darwin)
-    @linux_only push!(BinDeps.defaults, Binaries)
+    @compat @static is_linux() ? push!(BinDeps.defaults, Binaries) : nothing
```

Comment:
Thanks @tkelman, also we need to `pop!` just after the installation, i.e., `@BinDeps.install`, right?